test/WebJobs.Script.Tests/Scale/ScaleMonitorServiceTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.WebJobs.Script.Tests;
+using Moq;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Scale
+{
+    public class ScaleMonitorServiceTests
+    {
+        [Theory]
+        [InlineData(true, true)]
+        [InlineData(false, true)]
+        [InlineData(true, false)]
+        [InlineData(false, false)]
+        public async Task ScaleMonitorService_RegistersExpected(bool runtimeScaleMonitoringEnabled, bool targetBaseScalingEnabled)
```

Comment:
What is this test testing? ScaleMonitorService doesn't register anything. Does this belong in the SDK ScaleMonitorServiceTests suite with all the other tests?

Reply:
yes, renamed